### ISSUE TITLE
[BUG] Release 5.1.0 export "Components" was not found in 'design-react-kit'

### ISSUE THREAD

**Issue by sabato-galasso** [italia/design-react-kit#1080]
**Describe the bug**
Components not importable

**To Reproduce**
Import an Component of the  release 5.1.0

**Expected behavior**
export 'Button' (imported as 'Button') was not found in 'design-react-kit'

**Screenshots**

![Screenshot 2024-07-09 alle 17 47 15](https://github.com/italia/design-react-kit/assets/52424004/d6b2639a-592a-4148-b047-a7dc311364d9)

**Comment by sabato-galasso** [comment]
Workaround remove  "type": "module" in packages.json

**Comment by Virtute90** [comment]
I don't have this kind of problem with 5.1.0, have you tried putting `"type": "module"` in your `package.json`?

**Comment by sabato-galasso** [comment]
> I don't have this kind of problem with 5.1.0, have you tried putting `"type": "module"` in your `package.json`?

yes, I have the same problem, are you sure that type:module is the expected behavior?

**Comment by Fupete** [comment]
@Lorezz wdyt of this issue? cc @astagi

**Comment by Lorezz** [comment]
solved on  [create react app scaffold example](https://github.com/sabato-galasso/react-test)
